jnigen/example/pdfbox_plugin/example/macos/Flutter/ephemeral/Flutter-Generated.xcconfig

```diff
@@ -0,0 +1,11 @@
+// This is a generated file; do not edit or check into version control.
```

Comment:
Looks like we're missing a gititgnore somewhere.

Reply:
macos is not supported yet, so I'll just remove it. It was accidentally added after the pub version change + upgrade.